pkg/client/src/app/shared/hooks/useHasIdentity.ts

```diff
@@ -0,0 +1,20 @@
+import { Application } from "@app/api/models";
+import { useFetchIdentities } from "@app/shared/hooks/useFetchIdentities";
+
+interface IUseHasIdentity {
+  hasIdentity: (application: Application, kind: string) => boolean;
+  fetchIdentities: () => void;
+}
+
+export const useHasIdentity = (): IUseHasIdentity => {
+  const { identities, fetchIdentities } = useFetchIdentities();
```

Comment:
I think you could pass in the identities to this hook as a param rather than calling this here.